pgx/src/datum/time_stamp.rs

```diff
@@ -8,6 +8,13 @@ use time::{format_description::FormatItem, PrimitiveDateTime};
 
 #[derive(Debug)]
 pub struct Timestamp(time::PrimitiveDateTime);
+
+impl From<pg_sys::Timestamp> for Timestamp {
+    fn from(item: pg_sys::Timestamp) -> Self {
+        unsafe { Timestamp::from_datum(item as usize, false, pg_sys::TIMESTAMPOID).unwrap() }
```

Comment:
Seems these should be `TryFrom` given that you're unwrapping them?

Reply:
I'm actually not sure about this. Because I'm passing false for `is_null` I can never get a `None` back, so the unwrap is fine.

What I'm not sure about is what a `pg_sys::Timestamp` which actually is null (as opposed to an actual Option) would look like and how you'd get it?

Reply:
ZomboDB: Yah. That’ll never panic

I'm feeling more confident now!